js/scripts.js

```diff
@@ -0,0 +1,15 @@
+var a = prompt("Eneter number a");
+var b = prompt("Eneter number b");
+var value = (a * a) - (2 * a * b) + (b * b);
+alert("value is equal to  "+value)
+console.log("value is equal to  "+ value);
+if (+ value > 0 ) {
+	console.log('value is positive');
+} else if (+ value < 0 ) {
+	console.log('value is negative');
+}
+if (+value == 0) {
```

Comment:
it will be next condition for rest of your conditions, so better you add here `else if` instead of `if`

are you sure that your condition is correct? compare also type of `value` and `0`